Cards/Profiles/index.css

```diff
@@ -0,0 +1,95 @@
+.profile-block-container{
+  width:500px;
+  margin: 0 auto;
+}
+
+.upper-profile-container{
+  height:50%;
+  width:100%;
+  background-image: radial-gradient(#0080ff,#6593e5,#0f52ba);
+  z-index:-1;
+  background-repeat: no-repeat;
+  color:white;
+  text-align: center;
+}
+
+.profile-name{
+  font-size: 2.5rem;
+  font-weight: bold;
+  padding-top: 20px;
+  text-shadow: 1px 1px black;
+}
+
+.profile-work{
+  font-size:1.5rem;
+}
+.upper-profile-container img{
+  width: 150px;
+  height:150px;
+  border-radius: 50%;
+  margin-bottom: 20px;
+}
+.follow-btn{
+  background-image: linear-gradient(#3DF6AA,#1ED064);
+  border-radius: 20px;
+  width:30%;
+  margin: 0 auto;
+  color: white;
+  font-size: 30px;
+  font-weight: 700;
+  padding: 5px;
+  font-family: sans-serif;
+  text-shadow: 1px 1px 4px #000000;
+  text-decoration: none;
+  display: inline-block;
+  cursor: pointer;
+}
```

Comment:
Good Functionality for artists needs a bit of reduction in size. Adding two distinct buttons one for Insta and other for Facebook can be a better addon.

Reply:
Buttons will be distinguished by bg colors.
Green one was for fb. as there the background was already blue.
Newly added will be of insta with background as purple-pink